### DISCUSSION TITLE
How to convert an array from id to an array of objects?

### DISCUSSION THREAD

**Discussion by Fatonndev** [prisma/prisma#14913] [Q&A] [answered]
I've been thinking for a long time how to make such a connection, do you have any ideas?

```prisma
model User {
    id      Int      @id @default(autoincrement())
    ...
    groups  Group[]
}
```

```prisma
model Group {
    id       Int      @id @default(autoincrement())
    ...
    users    User[]  @relation(fields: [usersId], references: [id])
    usersId  Int[]
}
```

**Comment by austincrim** [comment]
Hey @Fatonndev 👋,

Could you elaborate on your goal here? I'm a bit unsure what you are looking for.

**Comment by Fatonndev** [comment, marked as answer]
I can't figure out how to use the array `users Id Int[]` and `@relation` to get the array `users User[]`. Throws an error that postgresql does not support many-to-many relations

**Reply by austincrim** [reply]
Can you share the code snippet that is triggering the error?

**Reply by Fatonndev** [reply]
An excerpt of the code is specified in the topic of the discussion

**Reply by austincrim** [reply]
Ah, I see now.

The link provided in the error message instructs you to use an [implicit many-to-many relation](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/many-to-many-relations#implicit-many-to-many-relations) instead.
```prisma
model User {
    id      Int      @id @default(autoincrement())
    groups  Group[]
}

model Group {
    id       Int      @id @default(autoincrement())
    users    User[]  
}
```

Does this approach work for you?

**Reply by Fatonndev** [reply]
It's works, thx

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
